src/OrleansAzureUtils/Providers/Storage/AzureTableStorage.cs

```diff
@@ -103,8 +103,7 @@ public Task Init(string name, IProviderRuntime providerRuntime, IProviderConfigu
             
             if (useJsonFormat)
             {
-                jsonSettings = new Newtonsoft.Json.JsonSerializerSettings();
-                jsonSettings.TypeNameHandling = Newtonsoft.Json.TypeNameHandling.All;
+                jsonSettings = OrleansJsonSerializer.SerializerSettings;
```

Comment:
This whole PR was just for this one line.
The property needs to start with capital letter, by convention, so this explain the rest.
Do you want em to minimize changes, to a bare minimal? can do.